src/screens/ChildrenScreen.js

```diff
@@ -1,158 +1,126 @@
-                ></Ionicons>
+            <View key={u.id}>
+              <View style={styles.container}>
+                {/* <Image source={{ uri:u.avatar }} /> */}
+                <Image
+                  style={styles.image}
+                  source={require("../../assets/icons/logo.png")}
+                />
+                <View style={styles.body}>
+                  <Text style={styles.name}>{u.name}</Text>
+                  <Text style={styles.birth}>BIRTH: {u.birth}</Text>
+                  <Text style={styles.birth}>LATEST: {u.latest}</Text>
+                </View>
               </View>
             </View>
           );
         })}
       </View>
-    );
-  };
-  return (
-    <View style={screen.container}>
-      <ScrollView>{renderHistories()}</ScrollView>
-      <Ionicons
-        name="ios-add-circle"
-        size={90}
-        color={colors.lightBlue}
-        style={styles.addIcon}
-        onPress={() =>
-          navigation.navigate("AddChildScreen", { screen: "AddChildScreen" })
-        }
-      ></Ionicons>
+      <View style={styles.foot}>
+        <TouchableOpacity onPress={() => navigation.navigate("AddChildScreen")}>
+          <Image
+            style={styles.addIcon}
+            source={require("../../assets/icons/add-profile.png")}
+          ></Image>
+        </TouchableOpacity>
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+  },
+  header: {
+    flex: 0.8,
+  },
+  foot: {
+    flex: 0.2,
+  },
   container: {
-    width: "95%",
-    marginLeft: "5%",
-    marginTop: "2%",
-    paddingBottom: "2%",
-    borderBottomColor: colors.lightBlue_button,
-    borderBottomWidth: 1,
+    height: moderateScale(96),
+    backgroundColor: colors.white,
+    flexDirection: "row",
+    marginBottom: scale(10),
   },
-  listStyle: { paddingRight: "12%" },
-  type: {
-    fontSize: 20,
+  name: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(16),
     fontWeight: "bold",
-    color: colors.lightBlue,
+    lineHeight: moderateScale(22),
+    margin: scale(5),
   },
-  date: {
-    fontSize: 15,
+  birth: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(12),
+    color: "#1F263E",
+    lineHeight: moderateScale(16.34),
+    margin: 5,
   },
-  note: { fontSize: 14 },
-  arrowIcon: {
+  image: {
     flex: 1,
-    alignItems: "center",
-    position: "absolute",
-    marginTop: "6%",
-    right: 0,
+    resizeMode: "center",
+    height: moderateScale(70),
+    width: moderateScale(70),
+    margin: scale(10),
+    justifyContent: "space-around",
+    borderRadius: 400 / 2,
   },
-  addIcon: {
+  body: {
+    flex: 2,
+    justifyContent: "center",
+  },
+  smallIcon: {
+    flexDirection: "column",
     flex: 1,
-    position: "absolute",
+  },
+  addIcon: {
     alignSelf: "flex-end",
-    bottom: 0,
-    right: 0,
+    width: moderateScale(70),
+    height: moderateScale(70),
+    marginRight: moderateScale(23),
```

Comment:
Use scale for width size/marginRight/marginLeft and verticalScale for height size/marginTop/marginbottom. Please check this and fix it on the other screens you created (EditChildrenScreen)